tests/adapters_t/sites_t/test_kubernetes.py

```diff
@@ -70,8 +70,13 @@ def setUp(self):
             args=["sleep", "3600"],
             name="testsite-089123",
             resources=client.V1ResourceRequirements(
-                requests={"cpu": "2", "memory": "400Mi"}
+                requests={"cpu": 2, "memory": 4000000000}
```

Comment:
Just for clarification. Internally the `convert_to` is used? So that this test also ensures, that it does the correct conversion?
Further it might make sense to not directly write `4000000000` but put the memory value from line 59 into a variable to calculate that value instead. What do you think?

Reply:
> Just for clarification. Internally the `convert_to` is used? So that this test also ensures, that it does the correct conversion?

Yes, it does.

> Further it might make sense to not directly write `4000000000` but put the memory value from line 59 into a variable to calculate that value instead. What do you think?

Fixed that in the latest commit.